tutorials/ui/size_and_anchors.rst

```diff
@@ -17,37 +15,53 @@ and mobile phones have different resolutions and aspect ratios.
 There are several ways to handle this, but for now, let's just imagine
 that the screen resolution has changed and the controls need to be
 re-positioned. Some will need to follow the bottom of the screen, others
-the top of the screen, or maybe the right or left margins.
+the top of the screen, or maybe the right or left borders.
+
+.. image:: img/ui_anchor_and_margins.webp
+
+This is done using the *margin* and *anchor* properties. Margins represent a
+distance in pixels relative to the top-left corner of the parent control or
+(in case there is no parent control) the viewport. Anchors adjust where the
+margin distances are relative *to*. We can think of them as a percentage of the
+parent's size, represented by a value between 0 and 1.0.
 
-.. image:: img/anchors.png
+:ref:`Containers <doc_gui_containers>` are used to compose the layout of control
+scenes, offering powerful layout and sizing functionality. The MarginContainer
+is where we typically set margins for its child controls. When using containers,
+controls will inherit the layout properties of their parent, removing the need
+to modify the anchors of a control.
+
+Layout Presets
+--------------
 
-This is done by editing the *margin* properties of controls. Each
-control has four margins: left, right, bottom, and top, which correspond
-to the respective edges of the control. By default, all of
-them represent a distance in pixels relative to the top-left corner of
-the parent control or (in case there is no parent control) the viewport.
+Each control has an individual anchor that can be adjusted from the beginning
+to the end of the parent. So the vertical (top, bottom) anchors adjust from 0
+(top of parent) to 1.0 (bottom of parent) with 0.5 being the center. The
+horizontal (left, right) anchors similarly adjust from left to right of the
+parent.
```

Comment:
Why is this mentioned in the section about presets? It should be explained above, in the section about the basics.